debugtools/DDR_VM/src/com/ibm/j9ddr/tools/StructureStubGenerator.java

```diff
@@ -225,10 +235,11 @@ private static void writeOffsetStub(PrintWriter writer, FieldDescriptor fieldDes
 		}
 	}
 
-	private static void writeOffsetInitializer(PrintWriter writer, StructureDescriptor structure) {
+	private void writeOffsetInitializer(PrintWriter writer, StructureDescriptor structure) {
 		Collections.sort(structure.getFields());
 		for (FieldDescriptor fieldDescriptor : structure.getFields()) {
-			if (PointerGenerator.getOffsetConstant(fieldDescriptor).equals(fieldDescriptor.getName())) {
+			if (getOffsetConstant(fieldDescriptor).equals(fieldDescriptor.getName())
+					&& !PointerGenerator.omitFieldImplementation(structure, fieldDescriptor)) {
 				String fieldName = fieldDescriptor.getName();
```

Comment:
Please move this declaration before the `if` statement so it can be reused in the conditon.